package.json

```diff
@@ -22,7 +22,8 @@
     "next:dev": "cd ./next-app && yarn && node ../uui-build/scripts/runNextApp.js && yarn dev",
     "postinstall": "cd server && yarn",
     "prepare": "husky install",
-    "process-icons": "node uui-build/scripts/processIcons.js",
+    "process-icons": "node uui-build/scripts/processIcons.js icons",
+    "process-system-icons": "node uui-build/scripts/processIcons.js system-icons",
```

Comment:
since we decided to not go with system icons, we can remove this command

Reply:
fixed